packages/client/src/App.sass

```diff
@@ -21,6 +21,7 @@ $white-20: rgba(255,255,255, 0.2)
 $grey-primary: #939393
 $grey-dark: #6D6D6D
 $grey-secondary: #DDDDDD
+$grey-light: #DDDDDD // Override bulma default to match designs
```

Comment:
The reason was that all of the `.border-light` classes were using `$grey-light` and `$grey-light` wasn't being used anywhere else so I thought it would keep the naming a bit more intuitive.

Reply:
revisited this, i the override is unnecessary. i'll just use $grey-secondary :)